app/components/shared/inputs/CodeInput.vue.ts

```diff
@@ -14,6 +14,9 @@ export default class CodeInput extends BaseInput<string, IInputMetadata> {
   @Prop({ default: '' })
   readonly value: string;
 
+  @Prop()
+  readonly title: string;
```

Comment:
I added the `title` prop in each Input component for backward capability with old code that added this field in the base class